### PR TITLE
Feature/deliverable upload

### PR DESCRIPTION
**Pull request by kitakaze-kan**
## Describe your changes
- Support and migrate for the new Ceramic data model `WorkCredential`
- Support deliverable data upload ( to ipfs with web3.storage)
- Fix some rendering errors

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? 
- [X] Will this be part of a product update? If yes, please write one phrase about this update.

## Reference
web3.storage
https://web3.storage/docs/

### CODE REVIEW

File: src/components/CVoxel/CVoxelDetailBox.tsx [hunk 1]
```diff
@@ -202,25 +204,25 @@ export const CVoxelDetailBox: FC<{}> = () => {
         </div>
 
         {/*deliverables*/}
-        {detailItem?.deliverable && (
+        {detailItem?.deliverables && (
```
Comment (0x-shanks) [inline src/components/CVoxel/CVoxelDetailBox.tsx:205]: 🙇‍♂️

**Review by 0x-shanks — APPROVED** [review]
LGTM!
I think you have checked the operation and I will approve. In my local environment, createDraftWighVerify gives me "signature is not valid", so I have not been able to check all the operations.

File: src/components/CVoxel/CVoxelDetailBox.tsx (hunk 1 above)
Comment (kitakaze-kan) [reply src/components/CVoxel/CVoxelDetailBox.tsx:205]: oh, so you can't make the new one?
hmm... ok , I'll check it again!

Comment (0x-shanks) [reply src/components/CVoxel/CVoxelDetailBox.tsx:205]: thanks! Maybe it's just my local environment.

Comment (kitakaze-kan) [reply src/components/CVoxel/CVoxelDetailBox.tsx:205]: I have checked again in my local and it seems working. so merged it.

**Pull request merged**